src/states/useAuthStore.tsx

```diff
@@ -24,6 +25,7 @@ const useAuthStore = create<IAuthStore>((set) => {
       // save userId, accessToken to LocalStorage
       const authDataToSave = { accessToken }
       LocalStorage.setItem('authData', JSON.stringify(authDataToSave))
+      setCookie('accessToken', accessToken)
```

Comment:
why cookie?

Reply:
Because we use SSR on the main page. But when we load the favorite things on the main page, we need to send my  access token contained to client side. In this case, server side doesn't have the access token, so the data we want doesn't come. But cookie can be server side too. This is why we can use cookie.